sources/AppBundle/Event/Model/Event.php

```diff
@@ -712,6 +717,24 @@ public function setWaitingListUrl($waitingListUrl)
         return $this;
     }
 
+    public function hasPricesDefinedWithVat()
+    {
+        return (bool) $this->hasPricesDefinedWithVat;
+    }
+
+    /**
+     * @param bool $hasPricesDefinedWithVat
+     *
+     * @return $this
+     */
+    public function setHasPricesDefinedWithVat($hasPricesDefinedWithVat)
```

Comment:
Pareil ici en PHP 7 :
```suggestion
    public function setHasPricesDefinedWithVat(bool $hasPricesDefinedWithVat)
```